test/unit/gateways/moneris_us_test.rb

```diff
@@ -22,6 +22,21 @@ def test_default_options
     assert_equal "qatoken", @gateway.options[:password]
   end
 
+  def test_successful_verify
+    @gateway.expects(:ssl_post).twice.returns(successful_purchase_response)
```

Comment:
I discussed with @ntalbott.  I had originally made an auth response, but it was identical to the purchase response so it seemed unnecessary.